server/app/views/style/BaseStyles.java

```diff
@@ -95,6 +95,13 @@ public final class BaseStyles {
   /** For use on an `input` of type "radio". */
   public static final String RADIO = CHECKBOX;
 
+  /** For use on a `select` element. */
+  public static final String SELECT = StyleUtils.joinStyles(BaseStyles.INPUT, "h-11.5");
+
+  /** For use on a `select` element. */
+  public static final String SELECT_WITH_ERROR =
```

Comment:
Could you update the javadoc for `SELECT_WITH_ERROR` to differentiate itself from `SELECT`?

Reply:
done